src/argilla_server/models/questions.py

```diff
@@ -12,17 +12,21 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-from typing import Any, Generic, List, Literal, Optional, Protocol, TypeVar, Union
+from typing import Any, Dict, Generic, List, Literal, Optional, Protocol, TypeVar, Union
 
 from argilla_server.enums import QuestionType, ResponseStatus
-from argilla_server.pydantic_v1 import BaseModel, Field
+from argilla_server.pydantic_v1 import BaseModel, Field, root_validator
 
 try:
     from typing import Annotated
 except ImportError:
     from typing_extensions import Annotated
 
 
+SPAN_QUESTION_RESPONSE_VALUE_MIN_ITEMS = 1
```

Comment:
We should accept an empty spans list, which means that no entities were found for the provided text 
```suggestion
SPAN_QUESTION_RESPONSE_VALUE_MIN_ITEMS = 0
```

Reply:
Done.